core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.css

```
@@ -0,0 +1,80 @@
+  position: relative;
+}
+.oppia-thumbnail-container img {
+  border-radius: 4px 4px 0 0;
+}
+.oppia-thumbnail-reset-button {
+  position: absolute;
+  right: -50px;
+  top: 0;
+}
+.oppia-thumbnail-uploader {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)